test/hotspot/jtreg/compiler/vectorapi/AllBitsSetVectorMatchRuleTest.java

```diff
@@ -63,6 +63,9 @@ public class AllBitsSetVectorMatchRuleTest {
     private static boolean[] mb;
     private static boolean[] mc;
     private static boolean[] mr;
+    private static long[] la;
+    private static long[] lb;
+    private static long[] lr;
```

Comment:
Can you reoder it a bit so that int[] and long[] test cases are grouped together?